sql/lease.go

```diff
@@ -151,11 +152,13 @@ func (s LeaseStore) Acquire(txn *client.Txn, tableID ID, minVersion DescriptorVe
 		}
 		return nil
 	})
```

Comment:
Isn't `roachpb.NewError(pErr.GoError())` equivalent to `pErr`? Why go through the extra conversions?